packages/app/src/api.ts

```diff
@@ -171,6 +180,126 @@ export function initializeApp(
   return newApp;
 }
 
+/**
+ * Creates and initializes a {@link @firebase/app#FirebaseServerApp} instance.
+ *
+ * The `FirebaseServerApp` is similar to `FirebaseApp`, but is intended for execution in
+ * server side rendering environments only. Initialization will fail if invoked from a
+ * browser environment.
+ *
+ * See
+ * {@link
+ *   https://firebase.google.com/docs/web/setup#add_firebase_to_your_app
+ *   | Add Firebase to your app} and
+ * {@link
+ *   https://firebase.google.com/docs/web/setup#multiple-projects
+ *   | Initialize multiple projects} for detailed documentation.
+ *
+ * @example
+ * ```javascript
+ *
+ * // Initialize an instance of `FirebaseServerApp`.
+ * // Retrieve your own options values by adding a web app on
+ * // https://console.firebase.google.com
+ * initializeServerApp({
+ *     apiKey: "AIza....",                             // Auth / General Use
+ *     authDomain: "YOUR_APP.firebaseapp.com",         // Auth with popup/redirect
+ *     databaseURL: "https://YOUR_APP.firebaseio.com", // Realtime Database
+ *     storageBucket: "YOUR_APP.appspot.com",          // Storage
+ *     messagingSenderId: "123456789"                  // Cloud Messaging
+ *   },
+ *   {
+ *    authIdToken: "Your Auth ID Token"
+ *   });
+ * ```
+ *
+ * @param options - `Firebase.AppOptions` to configure the app's services, or a
+ *   a `FirebaseApp` instance which contains the `AppOptions` within.
+ * @param config - `FirebaseServerApp` configuration.
+ *
+ * @returns The initialized `FirebaseServerApp`.
+ *
+ * @public
+ */
+export function initializeServerApp(
+  options: FirebaseOptions | FirebaseApp,
+  config: FirebaseServerAppSettings
+): FirebaseServerApp;
+
+export function initializeServerApp(
+  _options: FirebaseOptions | FirebaseApp,
+  _serverAppConfig: FirebaseServerAppSettings
+): FirebaseServerApp {
+  if (isBrowser()) {
+    // FirebaseServerApp isn't designed to be run in browsers.
+    throw ERROR_FACTORY.create(AppError.INVALID_SERVER_APP_ENVIRONMENT);
+  }
+
+  if (_serverAppConfig.automaticDataCollectionEnabled === undefined) {
+    _serverAppConfig.automaticDataCollectionEnabled = false;
+  }
+
+  let appOptions: FirebaseOptions;
+  if (_isFirebaseApp(_options)) {
+    appOptions = _options.options;
```

Comment:
we should move over settings like automaticDataCollectionEnabled, etc. if they've been defined on the app

Reply:
`automaticDataCollectionEnabled` is already accounted for. It is defined as a member of `FirebaseServerAppSettings`, which is part of the `_serverAppConfig` in this function.

Reply:
I wonder if we should default it to the value in FirebaseApp, if that was provided rather than FirebaseOptions

Reply:
We talked about this offline and we're not going to passively inherit the `automaticDataCollectionEnabled` value.